frontend/src/bundles/common/components/user-photo-uploader/user-photo-preview.tsx

```diff
@@ -0,0 +1,59 @@
+// import { UserProfileResponce } from 'shared/build/bundles/users/types/user-auth-response.type';
+// import { UserPhotoPlaceholder } from './user-photo-placeholder';
+// import { updateUserAvatar } from '~/bundles/profile/store/actions';
+
+import { 
+    // useAppDispatch, 
+    // useAppSelector, 
+    useCallback 
+} from '../../hooks/hooks';
+import styles from './styles.module.scss';
+
+interface UserPhotoProperties {
+    setIsModalOpen: React.Dispatch<React.SetStateAction<boolean>>;
+    currentPhoto: string;
+    setCurrentPhoto: React.Dispatch<React.SetStateAction<string>>;
+    handleImageChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const UserPhotoPreview: React.FC<UserPhotoProperties> = ({ setIsModalOpen, currentPhoto, setCurrentPhoto, handleImageChange }) => {
+
+    // const dispatch = useAppDispatch();
+    // const user = useAppSelector(({ auth }) => {
+    //     return auth.user as UserProfileResponce;
+    // });
+
+    const onEditClick = useCallback(() => {
```

Comment:
Change to have prefix handle for all
```suggestion
    const handleEditClick = useCallback(() => {
```

Reply:
Done